### PR TITLE
Auto complete tasks in Build #6

### PR DESCRIPTION
**Pull request by luzarek**
Open all Works in separate tabs

### CODE REVIEW

File: buildExtension.js [hunk 1]
```diff
@@ -1,19 +1,41 @@
 var getWorksFromBuild = () => {
 	var list = document.getElementsByClassName("dataRow");
-	var items = "";
+	var result = [];
```
Comment (mromaniszyn) [inline buildExtension.js]: var workInfos

File: buildExtension.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+			id: aNode.innerHTML,
+			name: currentDataCell.innerHTML,
+			href: aNode.href
+		};
+		result.push(workInfo);
+	}	
+	return result;
+};
+
+var addCopyToClipboardButton = () => {
```
Comment (mromaniszyn) [inline buildExtension.js]: var copyWorksToClipBoard

File: buildExtension.js [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
 		var iPlus = i + 1;
-		var itemString = iPlus + ". " + workId + " " + workName + "\n";
+		var itemString = iPlus + ". " + work.id + " " + work.name + "\n";
 		items += itemString;
 	}
-	
 	SFAA.copyText(items);
 };
 
+var addOpenAllButton = () => {
```
Comment (mromaniszyn) [inline buildExtension.js]: var openAllWorks

File: buildExtension.js [hunk 4]
```diff
@@ -23,21 +45,40 @@ var addBuildNameButtons = () => {
 	SFAA.createCopyButton(buildNameNode, 'Copy Build Name', 'Copy Build Name to clipboard', () => { return buildNameNode.innerHTML; });
 };
 
+var addButtonToListOfWorks = (cfg) => {
```
Comment (mromaniszyn) [inline buildExtension.js]: config

File: buildExtension.js [hunk 5]
```diff
@@ -23,21 +45,40 @@ var addBuildNameButtons = () => {
...
-			button.appendChild(buttonText);
-			button.id = "copyWorksToClipboardButton";
-			button.addEventListener("click", getWorksFromBuild);
-			
-			var tdElement = document.createElement("td");
-			tdElement.appendChild(button);
-			pbTitle.insertAdjacentElement("afterEnd", tdElement);
+			addButtonToListOfWorks({
+				buttonText: "Open All Works",
+				buttonId: "Open All Works In Separate Tabs",
```
Comment (mromaniszyn) [inline buildExtension.js]: It should be an ID - string without spaces. This look like tooltip text.

File: buildExtension.js (hunk 1 above)
Comment (luzarek) [reply buildExtension.js]: Done

File: buildExtension.js (hunk 2 above)
Comment (luzarek) [reply buildExtension.js]: Done

File: buildExtension.js (hunk 3 above)
Comment (luzarek) [reply buildExtension.js]: done

File: buildExtension.js (hunk 4 above)
Comment (luzarek) [reply buildExtension.js]: done

File: buildExtension.js (hunk 5 above)
Comment (luzarek) [reply buildExtension.js]: Added Title attribute

**Pull request merged**